### PR TITLE
[skip ci]Delete related Telegram group information

### PR DESCRIPTION
**Pull request by qwq233**
## 介绍 / Description
Given that the related group has been dissolved, has deleted related Telegram group information

## 更改内容 / Types of Changes

- [ ] 内核 Core
- [ ] 修复 Bugfix
- [ ] 新功能 New feature
- [ ] 增强/优化 Enhancement/optimization
- [x] 文档 Documentation
- [ ] UI界面 User interface

## 由这个PR解决或关闭的Issues / Issues Fixed or Closed by This PR

* 

## 检查列表 / Checklist

- [x] 我的代码符合本仓库的代码规范 My code follows the code style of this project
- [ ] 我已经测试了这些更改，它们可以正常工作，且不会破坏任何功能(或我可以解决) I have tested the changes and verified that they work and don't break anything (as well as I can manage).
- [x] 我已合并了对后续工作无意义的commit并确认不会对后续维护造成困扰 I have merged commits that are meaningless for follow-up work and confirmed that they will not cause damage to follow-up maintenance

### CODE REVIEW

**Review by JasonKhew96 — APPROVED** [review]

**Review by singleNeuron — CHANGES_REQUESTED** [review]
把telegram改成机器人吧

File: app/src/main/java/nil/nadph/qnotified/activity/DonateActivity.java [hunk 1]
```diff
@@ -117,7 +117,6 @@ public void onCheckedChanged(final CompoundButton buttonView, boolean isChecked)
         ll.addView(subtitle(this, "最后,谢谢你的支持"));
         ll.addView(subtitle(this, "by"));
         ll.addView(newListItemButton(this, "更新频道", null, "@QNotified", clickToUrl("https://t.me/QNotified")));
```
Comment (singleNeuron) [inline app/src/main/java/nil/nadph/qnotified/activity/DonateActivity.java:119]:
```suggestion
        ll.addView(newListItemButton(this, "更新频道", null, "@QNotified", clickToUrl("https://t.me/QNotified_bot")));
```

**Review by singleNeuron — CHANGES_REQUESTED** [review]

File: app/src/main/java/nil/nadph/qnotified/activity/DonateActivity.java [hunk 2]
```diff
@@ -116,8 +116,7 @@ public void onCheckedChanged(final CompoundButton buttonView, boolean isChecked)
         ll.addView(subtitle(this, "本软件首发地为 https://github.com/ferredoxin/QNotified (求star/issue/pull request)"));
         ll.addView(subtitle(this, "最后,谢谢你的支持"));
         ll.addView(subtitle(this, "by"));
-        ll.addView(newListItemButton(this, "更新频道", null, "@QNotified", clickToUrl("https://t.me/QNotified")));
-        ll.addView(newListItemButton(this, "交流群", null, "@QNotifiedChat", clickToUrl("https://t.me/QNotifiedChat")));
+        ll.addView(newListItemButton(this, "更新频道", null, "@QNotified", clickToUrl("https://t.me/QNotified_bot")));
```
Comment (singleNeuron) [inline app/src/main/java/nil/nadph/qnotified/activity/DonateActivity.java:119]:
```suggestion
        ll.addView(newListItemButton(this, "更新频道", null, "@QNotified", clickToUrl("https://t.me/QNotified")));
```

File: app/src/main/java/nil/nadph/qnotified/activity/DonateActivity.java [hunk 3]
```diff
@@ -116,8 +116,7 @@ public void onCheckedChanged(final CompoundButton buttonView, boolean isChecked)
...
         ll.addView(newListItemButton(this, "Telegram", "点击私信", "Auride", clickToUrl("https://t.me/Auride")));
```
Comment (singleNeuron) [inline app/src/main/java/nil/nadph/qnotified/activity/DonateActivity.java:120]:
```suggestion
        ll.addView(newListItemButton(this, "Telegram", "点击私信", "客服机器人", clickToUrl("https://t.me/QNotified_bot")));
```

**Review by singleNeuron — CHANGES_REQUESTED** [review]

**Review by singleNeuron — APPROVED** [review]

**Comment by huajijam** [comment]
Merge Commit PLZ

**Comment by singleNeuron** [comment]
> Merge Commit PLZ

u can u up

**Comment by qwq233** [comment]
> Merge Commit PLZ

working on this
The master branch of my fork repository is a bit messy and has been extensively modified by me. May be troublesome

**Comment by qwq233** [comment]
> > Merge Commit PLZ
> 
> working on this
> The master branch of my fork repository is a bit messy and has been extensively modified by me. May be troublesome

done

**Comment by huajijam** [comment]
好耶

**Pull request merged**